### PR TITLE
Update to reflect recent discussions

### PR DESCRIPTION
**Pull request by CBonnell**
Incorporate conclusions reached during November and December validation-sc meetings:
- https://lists.cabforum.org/pipermail/validation/2021-November/001728.html
- https://lists.cabforum.org/pipermail/validation/2021-December/001738.html
- https://lists.cabforum.org/pipermail/validation/2021-December/001739.html
- the F2F validation-sc meeting (public minutes not yet available)

It was agreed (https://lists.cabforum.org/pipermail/validation/2021-November/001728.html) that the requirement to make the CABF policy OID the first policyIdentifier should be moved to another ballot. Given the drastic changes in formatting adjusting the current language would create, I haven't proposed concrete language for that yet.

### CODE REVIEW

**Comment by sleevi** [comment]
OK, so since this is an uber-PR, combining many different changes (hooray for separate commits), I'll go comment on individual commits. Ideally, we'll be able to rebase-and-merge at the end, unless you'd like to use branch-and-PR for the individual commits.

File: docs/BR.md [hunk 1]
```diff
@@ -2751,30 +2719,10 @@ Table: `GeneralName` requirements for the `base` field
...
-| `otherName`     | SHOULD NOT  | See following table.     | See following table.  | See following table.           |
+| `rfc822Name`    | MAY  | The CA MAY constrain to a mailbox, a particular host, or any address within a domain, as specified within [RFC 5280, Section 4.2.1.10](https://tools.ietf.org/html/rfc5280#section-4.2.1.10). For each host, domain, or Domain portion of a Mailbox (as specified within [RFC 5280, Section 4.2.1.6](https://tools.ietf.org/html/rfc5280#section-4.2.1.6)), the CA MUST confirm that the Applicant has registered the domain or has been authorized by the domain registrant to act on the registrant's behalf. See [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control). | If at least one `rfc822Name` instance is present in the `permittedSubtrees`, the CA MAY indicate one or more mailboxes, hosts, or domains to be excluded. | If no `rfc822Name` instance is present in the `permittedSubtrees`, then the CA MAY include a zero-length `rfc822Name` to indicate no mailboxes are permitted. |
+| `otherName`     | MAY   | -                        | -                     |
```
Comment (sleevi) [inline docs/BR.md:2721]: Both of these remain SHOULD NOT, not MAY, under the existing restrictions of the BRs today (7.1.2.4)

File: docs/BR.md [hunk 2]
```diff
@@ -2104,7 +2104,7 @@ CAs SHALL NOT include additional names unless the CA is aware of a reason for in
 | \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
 | \ \ \ \ `issuer`               | MUST be byte-for-byte identical to the `subject` field of the Issuing CA. See [Section 7.1.4.1](#7141-name-encoding) |
 | \ \ \ \ `validity`             | |
-| \ \ \ \ \ \ \ \ `notBefore`    | __**TBD**__ |
+| \ \ \ \ \ \ \ \ `notBefore`    | A value within 48 hours of the certificate signing operation. |
```
Comment (sleevi) [inline docs/BR.md:2107]: I realize this was from https://lists.cabforum.org/pipermail/validation/2021-November/001728.html but I didn't see any explanation/rationale?

Unlike CA signing, there's no need to backdate here, and this seems to violate the existing BRs' 9.6.1

Was the only concern to resolve the precertificate question? Just making sure I can try to address this correctly / offer suggestions here.

**Review by sleevi — COMMENTED** [review]
Thanks for doing this!

I'm not sure how well these comments will carry over, because they're on individual commits. We could try breaking them out into individual PRs (I believe if you git branch each commit, and then set their appropriate upstream, GitHub is supposed to be smart enough), but happy to continue iterating through this.

You may want to rebase -i to squash some of the fixups (like the dangling reference) into their appropriate section.

I notice you included a Python file for renumbering sections. That should be a separate commit and PR, don't you think?

File: docs/BR.md [hunk 3]
```diff
@@ -1768,17 +1768,16 @@ If the CA asserts compliance with these Baseline Requirements, all certificates
       * Cross Certificates
         * [Section 7.1.2.2 - Cross-Certified Subordinate CA Certificate Profile](#7122-cross-certified-subordinate-ca-certificate-profile)
       * Technically Constrained CA Certificates
-        * [Section 7.1.2.3 - Technically-Constrained Non-TLS Subordinate CA Certificate Profile](#7123-technically-constrained-non-tls-subordinate-ca-certificate-profile)
-        * [Section 7.1.2.4 - Technically-Constrained Precertificate Signing CA Certificate Profile](7124-technically-constrained-precertificate-signing-ca-certificate-profile)
-        * [Section 7.1.2.5 - Technically-Constrained TLS Subordinate CA Certificate Profile](#7125-technically-constrained-tls-subordinate-ca-certificate-profile)
-      * [Section 7.1.2.6 - TLS Subordinate CA Certificate Profile](#7126-tls-subordinate-ca-certificate-profile)
-  * [Section 7.1.2.7 - Subscriber (End-Entity) Certificate Profile](#7127-subscriber-server-certificate-profile)
```
[the remark this answers is not in the document]
Comment (ryancdickson) [reply docs/BR.md:1775]: @sleevi - check out the [F2F minutes](https://wiki.cabforum.org/meeting_55_minutes#validation_subcommittee_progress_report)

**Comment by sleevi** [comment]
@CBonnell Thanks for this.

I've updated https://github.com/sleevi/cabforum-docs/pull/36 with new commits related to this
* ✅  **Adopted** https://github.com/sleevi/cabforum-docs/pull/39/commits/04c55a4cdf2f6ea068bd1f743a83b60def34dcae - Adopted in https://github.com/sleevi/cabforum-docs/pull/36/commits/dd5e8fc2705ab73b42d93296d25cf56f2509c851
* ✅  **Adopted** https://github.com/sleevi/cabforum-docs/pull/39/commits/816ad7aa79cbfc1315561590d89ed7a9fd076b97 - In https://github.com/sleevi/cabforum-docs/pull/36/commits/a73dd6da1688dd6338ea0c85b190952b08bf21c4
* ✅  **Adopted** https://github.com/sleevi/cabforum-docs/pull/39/commits/41cb3063b41af69615bba5410279396994d2ebc0 - Should be adopted in https://github.com/sleevi/cabforum-docs/pull/36/commits/feeef8ad5bfb56d0b4263882b91c0a62e8428398
* 👍  **Partially Adopted** https://github.com/sleevi/cabforum-docs/pull/39/commits/57844af5ae1e9a3fd59efaa65813e30d4fab8a7b - OCSP Responders still prohibit CPs, but https://github.com/sleevi/cabforum-docs/pull/36/commits/d369fa503c5b379e8e7cb19aed0f19e83a67b1f8 should takle the AKI/SKI element?
* 👍  **Partially Adopted** https://github.com/sleevi/cabforum-docs/pull/39/commits/0bb9cdd2bf33968a7301f68a5fe667bfa2d90a16 - Overhauled `srvNames` in https://github.com/sleevi/cabforum-docs/pull/36/commits/74c89be1716a1d1c70e2fc98a7f7878d5dd65f91 to align with Mozilla policy updates (although 1.1 of Moz Policy still lists srvNames)
* 👍  **Partially Adopted** https://github.com/sleevi/cabforum-docs/pull/39/commits/2131f9451e867c14f31f5473c38a1254e84aedbb - Tackled in https://github.com/sleevi/cabforum-docs/pull/36/commits/037b36cf68313dd8a3a5d76e2a70aa9bbd71760b by changing both rules to being subject name related
* ❌  **Not Adopted** https://github.com/sleevi/cabforum-docs/pull/39/commits/a427efc4e6e0230244279a6d0c3c3fbfbb9a9bf4 - Kept the profile for non-intermediate CAs. However, attempted to clarify based on the discussion, in https://github.com/sleevi/cabforum-docs/pull/36/commits/45f634bfd8c1d8c9d0b7544aabf423820fd1ac11 (which specifies the EKUs fairly liberally) and https://github.com/sleevi/cabforum-docs/pull/36/commits/90d16e177b1b0ef5bbbaccaf9470757132b08919 (which fixed up a bad link and clarified the Certificate Policies). As explained, this tries to bring into alignment with RFC 5280, and the existing BRs around policies, but in a hopefully permissive way.
* ❌  **Not Adopted** https://github.com/sleevi/cabforum-docs/pull/39/commits/d7272d313ac59e62000d6110b62f64699ff21d2e - As discussed on the last profiles review

**Comment by sleevi** [comment]
(Closing this out, with hopefully everything addressed that was planned)

**Pull request closed without merging**